### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by csanders-git**
initial version, please provide feedback as desired.

### CODE REVIEW

**Comment by fzipi** [comment]
I think this is fantastic.

**Comment by fzipi** [comment]
@dune73 @lifeforms Do you think we need to add something here?

@csanders-git I guess we can create a GPG key for that email address, and begin to use it for signing releases also. What do you think?

**Comment by csanders-git** [comment]
I know there was discussion on signing releases, not sure where that went. but yeah --if we create said key signing releases should be a breeze.

**Comment by fzipi** [comment]
Exactly. So my proposal is to create `security@coreruleset.org` as a GPG key, and publish it for:
- signing releases
- receiving encrypted secure communications

**Comment by csanders-git** [comment]
@lifeforms we don't already have said key right?

**Comment by lifeforms** [comment]
@csanders-git no not yet!

**Comment by fzipi** [comment]
@lifeforms @csanders-git @dune73 Well, this is the moment where you guys generate the key. I can help if you want me to.

**Comment by lifeforms** [comment]
I generated a key. The public key is at https://coreruleset.org/security.asc

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| 3.0.x     | :x:                |
+
+## Reporting a Vulnerability
+
+We strive to make the OWASP ModSecurity CRS accessible to a wide audience of beginner and experienced users. 
+We are interested in hearing any bug reports, false positive alert reports, evasions, usability issues, and suggestions for new detections.
+These types of non-vulnerability related issues should be submitted via Github. 
+Please include your installed version and the relevant portions of your audit log.
+
+If you’ve found a false negative/bypass, please responsibly disclose the issue by sending an email to security@coreruleset.org.
```
Comment (fzipi) [inline SECURITY.md]: @csanders-git Should we add something here related to the key created?

**Comment by fgsch** [comment]
@lifeforms Great news! 
Have you uploaded it to any key server?

**Comment by fzipi** [comment]
I've uploaded the key to a couple servers already.

**Comment by fzipi** [comment]
Can the travis checks be run again?

**Review by fzipi — COMMENTED** [review]
Rebased and added text with GPG key and fingerprint.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+OWASP CRS has two major types of releases, Major releases (2.x.x, 3.x.x, 4.x.x)
```
Comment (fgsch) [inline SECURITY.md]: can we remove the first major? Too many major.

Comment (fgsch) [reply SECURITY.md]: Also, can we use real life examples? e.g. 4.x.x doesn't exist yet.

**Comment by fzipi** [comment]
Changing the text, and reviewing the wiki, I think we should change the naming we are using now and go for [semver](https://semver.org). Added the topic to the meeting agenda.

In our versioning policy we call version MAJOR and POINT releases, and I think semver has a better and clearer specification for this:

*    MAJOR version when you make incompatible API changes,
*    MINOR version when you add functionality in a backwards compatible manner, and
*    PATCH version when you make backwards compatible bug fixes.

**Comment by fgsch** [comment]
@fzipi I totally agree.

File: SECURITY.md (hunk 2 above)
Comment (dune73) [reply SECURITY.md]: Yes, I'd leave the 2.x.x and the 4.x.x away and just define 3.1.x and 3.2.x as major releases.

**Review by dune73 — COMMENTED** [review]
semver defines "MINOR version when you add functionality in a backwards compatible manner". 

Transporting this to our setting, this is not how we do things. The change from 3.1 to 3.2 is not backward compatible. It's potentially disruptive and if we start to use this wording, we are potentially misinforming users.

So I think 3.1 and 3.2 are major versions for us.

Just like Apache where 2.2 and 2.4 are major releases. 2.4.39 is minor release.

**Comment by fzipi** [comment]
I'm guessing we can adapt for now to this versioning and plan this for the future?

Using semantic versioning makes a lot of sense.

**Comment by fzipi** [comment]
@fgsch @dune73 Removed the text about 2.x.x and 4.x.x. But leaved the text as it was, to be consistent with the release policy.

We should extend the discussion about versions in the next community chat, but we can merge this one now.

**Comment by dune73** [comment]
Sorry, but I think the description of the versioning is still not crystal clear.

How about this?

```
OWASP CRS has two major types of releases, Major releases (3.0.0, 3.1.0, 3.2.0 etc.) and point releases (3.0.1, 3.0.2 etc.).
```


P.S. But I agree on the discussion of this topic in the chat.

**Comment by fzipi** [comment]
Reviewing the release policy, we are lucky enough to not mention numbers there. So I guess this could be changed without impact.

Another thing that we were discussing offline is this part: `If you’ve found a false negative/bypass, please responsibly disclose the issue by sending an email to `.

I guess we can agree that this seems a bad idea.

So, now the question is: what is a security issue for us in our context? Is this another topic for the chat?

**Comment by dune73** [comment]
Why do you think this is a bad idea?

And what is the bad idea exactly: The disclosure via email, or the sentence in our text?

**Comment by fzipi** [comment]
I think the sentence does not reflect exactly what we should do.

As an example, we have lots of false positives/bypasses documented in our issues list. I guess that ones should keep going there.

The ReDOS problems category we may want to receive them by email first, not as an issue, nor a CVE.

But I guess then the question is: what is a security issue in our context? I think the ReDOS, for sure. And bypasses/false negatives not 100%, because there will always be some by the nature of blacklisting.

**Comment by dune73** [comment]
You are right. We need a better wording.

**Comment by fzipi** [comment]
I'm going to reword based on what we talked on our meeting yesterday.

**Comment by fzipi** [comment]
@dune73 @csanders-git @fgsch Tried to add all things discussed in the chat. Please review.

**Comment by dune73** [comment]
I think that's a decent wording now and I am happy with it for the time being. If we want to improve / extend it in the future, then that should not be a problem.

**Comment by fzipi** [comment]
Shall we merge then?

**Pull request merged**